pkg/debug/scripts/collect-info.sh

```diff
@@ -5,10 +5,10 @@
 #
 
 # Script version, don't forget to bump up once something is changed
-VERSION=6
+VERSION=7
 
 # Add required packages here, it will be passed to "apk add".
-PKG_DEPS="procps dmidecode iptables dhcpcd"
+PKG_DEPS="procps tar dmidecode iptables dhcpcd"
```

Comment:
In your commit message: "This is correct **which** one exception" -> "This is correct **with** one exception

Reply:
Yep, thanks. Will fix